### PR TITLE
[ElmSharp] Added vd theme file path in ThemeOverlay for product tv

### PR DESCRIPTION
**Pull request by arosis78**
### Description of Change ###
Added vd theme file path in ThemeOverlay for product tv

### API Changes ###
None

### CODE REVIEW

File: src/ElmSharp/ElmSharp/Elementary.cs [hunk 1]
```diff
@@ -49,6 +49,7 @@ public enum FocusAutoScrollMode
     public static class Elementary
     {
         private const string _themeFilePath = "/usr/share/elm-sharp/elm-sharp-theme.edj";
+        private const string _vdThemeFilePath = "/usr/share/elm-sharp/vd-elm-sharp-theme.edj";
```
Comment (rookiejava) [inline src/ElmSharp/ElmSharp/Elementary.cs:52]: What abour `tvTheme` instead of `vdTheme`?

File: src/ElmSharp/ElmSharp/Elementary.cs [hunk 2]
```diff
@@ -236,9 +237,13 @@ public static void Run()
...
         public static void ThemeOverlay()
         {
-            if (!Window.IsPreloaded && File.Exists(_themeFilePath))
+            if (!Window.IsPreloaded)
             {
-                AddThemeOverlay(_themeFilePath);
+                if (File.Exists(_themeFilePath))
+                    AddThemeOverlay(_themeFilePath);
+
+                if (File.Exists(_vdThemeFilePath))
```
Comment (rookiejava) [inline src/ElmSharp/ElmSharp/Elementary.cs:245]:
```suggestion
                if (File.Exists(Elementary.GetProfile() == "tv" && _tvThemeFilePath))
```

File: src/ElmSharp/ElmSharp/Elementary.cs (hunk 1 above)
Comment (arosis78) [reply src/ElmSharp/ElmSharp/Elementary.cs]: Done

File: src/ElmSharp/ElmSharp/Elementary.cs (hunk 2 above)
Comment (arosis78) [reply src/ElmSharp/ElmSharp/Elementary.cs]: Done

**Review by rookiejava — APPROVED** [review]

**Pull request merged**